README.md

```diff
@@ -69,59 +72,73 @@
 
 ## Installation
 
-The easiest way to install `pyani` is to use `pip3` or `bioconda`:
+### IMPORTANT NOTICE
 
-### `pip3`
+**THIS REPOSITORY DEFAULT BRANCH CONTAINS A DEVELOPMENT VERSION OF `PYANI` (v0.3+). IT HAS A DIFFERENT COMMAND-LINE INTERFACE THAN THE STABLE `PYANI` VERSION (v0.2.x).**
 
-```bash
-pip3 install pyani
-```
+### Documentation for stable version `pyani` v0.2.x
+
+- [`pyani` v0.2.x Documentation](./README_v_0_2_x.md)
+
+**IF YOU WISH TO INSTALL THE STABLE VERSION OF `PYANI`, PLEASE FOLLOW THESE INSTRUCTIONS FOR INSTALLING v0.2.x**
+
+The easiest way to install `pyani` v0.2 is to use `conda` or  `pip`. `conda` is recommended for the simplest installation of third-party tool dependencies (`mummer` and `BLAST`/`BLAST+`).
+
+### `conda`
 
-From version 0.1.3.2 onwards, this should also install all the required Python package dependencies. Prior to this version (i.e. 0.1.3.1 and earlier), you can acquire these dependencies with `pip -r`, and pointing at `requirements.txt` from this repository:
+You will need to install the `bioconda` channel, following instructions at [https://bioconda.github.io/user/install.html](https://bioconda.github.io/user/install.html). Then, to create a new environment for `pyani` and install the program, issue the following command:
 
 ```bash
-pip3 install -r requirements.txt
-pip3 install -r requirements-pip.txt
+conda create --name pyani_env python=3.8 -y
+conda activate pyani_env
+conda install pyani
 ```
 
-### `bioconda`
+### `pip`
 
-With a working `anaconda` installation, install the `bioconda` and `conda-forge` channels:
+`pip` will install `pyani` and its Python dependencies, but not the third-party tools.
 
 ```bash
-conda config --add channels defaults
-conda config --add channels bioconda
-conda config --add channels conda-forge
+pip3 install pyani
 ```
 
-Then install `pyani`:
+### Third-party tools
+
+Three alignment packages are required, to use all of `pyani`'s methods: `mummer`, `BLAST+`, and legacy `BLAST`.
```

Comment:
Is it worth stating that if they only want to use some of `pyani`'s methods, they could only install the ones they want?

(Assuming, of course, that doesn't break something.)

Reply:
Maybe for clarity, yes. I wouldn't expect it to break anything.